CefSharp/CefSharp.csproj

```diff
@@ -93,6 +93,7 @@
     <Compile Include="DefaultResourceHandlerFactory.cs" />
     <Compile Include="DependencyChecker.cs" />
     <Compile Include="IJavascriptCallback.cs" />
+    <Compile Include="CEFCursorType.cs" />
```

Comment:
File name doesn't seem consistent with class name.  Shouldn't it be `CefCursorType.cs`?